index.html

```diff
@@ -67,121 +67,84 @@ <h1 style="margin-top: .15em;">
-    </div>
-
-    <div class="col-md-3">&nbsp;10:10am</div>
-    <div class="col-md-9">
-      <b>When Can we Ignore Missing Data in Model Training</b><br/>
-      <i>Cheng Zhen (Oregon State University), Amandeep Sing Chabada (Oregon State University), Arash Termehchy (Oregon State University)</i>
-    </div><br/>  
-    <br/><br/>&nbsp;<br/>
-
-    <div class="col-md-3"><b>&nbsp;10:30am - 11:00am</b></div><div class="col-md-9"><b>Break</b></div>
-
-    <br/>
-    <div class="col-md-12"></div><br/>
-    <div class="col-md-3"><b>&nbsp;11:00am - 12:20am</b></div><div class="col-md-9"><b>Session 2</b> - ML Optimization (Chair: Madelon Hulsebos)</div><br/>  
-    <div class="col-md-12"></div>
-
-    <div class="col-md-3">&nbsp;11:00am</div>
-    <div class="col-md-9">
-      <b>Using Pipeline Performance Prediction to Accelerate AutoML Systems</b><br/>
-      <i>Haoxiang Zhang (New York University), Roque Enrique López Condori (New York University), Aécio Santos (New York University), Jorge H Piazentin Ono (NYU), Aline Bessa (New York University), Juliana Freire (New York University)</i>
-    </div><br/>
-
-    <div class="col-md-3">&nbsp;11:20am</div>
-    <div class="col-md-9">
-      <b>P2D: A Transpiler Framework for Optimizing Data Science Pipelines</b><br/>
-      <i>Yordan Grigorov (Technische Universität Berlin), Haralampos Gavriilidis (Technische Universität Berlin), Sergey Redyuk (TU Berlin), Kaustubh Beedkar (IIT Delhi), Volker Markl (Technische Universität Berlin)</i>
-    </div><br/>  
-
-    <div class="col-md-3">&nbsp;11:40am</div>
-    <div class="col-md-9">
-      <b>Teaching Blue Elephants the Maths for Machine Learning</b><br/>
-      <i>Clemens Ruck (TUM); Maximilian E Schüle (University of Bamberg)*	1</i>
+    
+    <!-- Keynote -->
+    <div class="col-md-3"><b>9:00am - 10:00am</b></div>
+    <div class="col-md-9"><b>Opening Remarks and Keynote Address</b><br/>
+    <i>Speaker: <a href="https://engineering.nyu.edu/faculty/juliana-freire" target="_blank">Juliana Freire</a> (New York University)</i></div><br/>
+    
+    <!-- Break -->
+    <div class="col-md-3"><b>10:00am - 10:30am</b></div>
+    <div class="col-md-9"><b>Break</b></div><br/>
+
+    <!-- Session 1: Data Preparation and Pipelining -->
+    <div class="col-md-3"><b>10:30am - 11:20am</b></div>

```

Comment:
Might help authors (and attendees?) to see exact start times per talk (although they could be inferred)

Reply:
Good idea I can do this today